frontend/app/src/components/asset-manager/managed/ManagedAssetForm.vue

```diff
@@ -63,6 +65,21 @@ const isEvmToken = computed<boolean>(() => get(assetType) === EVM_TOKEN);
 
 const { allEvmChains } = useSupportedChains();
 
+const selectedAsetType = computed<SelectOption | undefined>({
+  get: () => !get(assetType) ? undefined : ({ key: get(assetType), label: toSentenceCase(get(assetType)) }),
+  set: value => set(assetType, value?.key),
+});
```

Comment:
Like I mentioned on discord I don't think this is a good approach, this wrapping seems really unnecessary. Can't we change the component to pass the `key-attr` value to the v-model directly?

Reply:
applying the change on the lib.